frontend/src/lib/k8s/grpcRoute.ts

```diff
@@ -0,0 +1,29 @@
+import { GatewayParentReference } from './gateway';
+import { KubeObject, KubeObjectInterface } from './KubeObject';
+
+export interface KubeGRPCRoute extends KubeObjectInterface {
+  spec: {
+    parentRefs: GatewayParentReference[];
+    [key: string]: any;
+  };
+}
+
+class GRPCRoute extends KubeObject<KubeGRPCRoute> {
+  static kind = 'GRPCRoute';
+  static apiName = 'grpcroutes';
+  static apiVersion = 'gateway.networking.k8s.io/v1beta1';
```

Comment:
Should this just be just gateway.networking.k8s.io/v1? https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1

If we need to add both, this can be done like this?
```
  static apiVersion = ['gateway.networking.k8s.io/v1', 'gateway.networking.k8s.io/v1beta1'];
```